tests/client/test_client.py

```diff
@@ -173,22 +173,3 @@ def test_merge_url():
 
     assert url.scheme == "https"
     assert url.is_ssl
-
-
-@pytest.mark.asyncio
-async def test_elapsed_delay(server):
-    url = server.url.copy_with(path="/slow_response/100")
-    async with httpx.Client() as client:
-        response = await client.get(url)
-    assert response.elapsed.total_seconds() == pytest.approx(0.1, rel=0.2)
```

Comment:
Can we maybe keep this one and assert that the number of seconds is between 0 and 0.5, manually? I don’t think we actually access response.elapsed anywhere else in the test suite.